### PR TITLE
ROC-3481: on PDFs total amount should not reflect interests

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-3481

### Change description ###
On PDF documents we don't want to display total amount with interest. 
On page interests should be added.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java [hunk 1]
```diff
@@ -56,7 +56,6 @@ public ClaimContent createContent(Claim claim) {
                 claim.getIssuedOn(),
                 claim.getIssuedOn()
             );
-            totalAmountComponents.add(interestContent.getAmountRealValue());
```
Comment (geek-soft) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java]: That change impacts sealed claim copy. @yarpo please make sure business (Art and Mark) understand that.

IMO `Claim amount: / Total:` without interest is less readable especially that section contains interest information.

Maybe more appropriate solution would be publishing PDF's into document store.

Comment (yarpo) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java]: I believe publishing would be an interesting solution. 

I agree that "total amount" which is not "so total" is misleading.

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java]: @yarpo I think we still need to add interest in sealed claim, but when the copy is downloaded it should not add daily interest  based on the days passed. In short no matter when user downloads the pdf copy it should reflect the same amounts as when sealed claim was generated. 

Agree with @geek-soft If we have `document-store` enabled/working probably better to store sealed claim there, and later retrieve it from there for copy.

**Comment by yarpo** [comment]
New PR:
https://github.com/hmcts/cmc-claim-store/pull/303

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java (hunk 1 above)
Comment (yarpo) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimDataContentProvider.java]: Please see:
https://github.com/hmcts/cmc-claim-store/pull/303

**Pull request closed without merging**